tutorials/scripting/gdscript/gdscript_basics.rst

```diff
@@ -1676,22 +1707,24 @@ so it won't generate infinite recursion and saves you from explicitly declaring
             changed.emit(value)
             warns_when_changed = value
 
-This backing member variable is not created if you don't use it.
```

Comment:
I thought this was an unnecessary clarification related to the implementation. Also, if I'm not mistaken, this is not true, the variable will take space in the `members` vector even if its value is not used.